core/src/main/scala/com/mesosphere/package.scala

```diff
@@ -0,0 +1,51 @@
+package com
```

Comment:
Should this be really part of core? I would not be surprised if the client would use this as well.

Reply:
Let's not create another module :) I am fine with this being in core for now :)